README.md

```diff
@@ -1,5 +1,11 @@
 # MOXA AIG Series OpenAPI Documents
 
+## AIG 501 Series
+| Version | Date | REST API  | Changelog | Latest |
+|  ----  | ----  | ----  | ----  | ----  |
+| V1.5| 2023-08 |- [Core](https://TPE-TIGER.github.io/AIG501/V1.3.0/core/#)<br />- [Device](https://TPE-TIGER.github.io/AIG501/V1.3.0/device/#)<br />- [Cloud](https://TPE-TIGER.github.io/AIG501/V1.3.0/cloud/#)<br />- [Taghub](https://TPE-TIGER.github.io/AIG501/V1.3.0/taghub/#)<br />- [Function](https://TPE-TIGER.github.io/AIG501/V1.3.0/function/#)<br />- [ModbusMaster](https://TPE-TIGER.github.io/AIG501/V1.3.0/modbusmaster/#) | [Changelog](https://github.com/TPE-TIGER/TPE-TIGER.github.io/blob/main/AIG501/CHANGELOG.md) | V |
```

Comment:
@MaWeiChi  The version `V1.5` and the date `2023-08` apparently are not correct.